lib/PreferenceList.cc

```diff
@@ -47,6 +48,40 @@ PrefListElement PreferenceList::at(SizeType index) const {
     return pref_list_.at(index);
 }
 
+void PreferenceList::set_ties(RankType rank, PreferenceList p){
+    SizeType size = p.size();
+    cur_rank_++;
```

Comment:
++cur_rank. Why is this named cur_rank_ and not currank like in a different function. Naming should be consistent.

Reply:
This was named already. It is not a local variable, but a private attribute.